rids/network_capture.py

```diff
@@ -14,74 +14,112 @@
 
 
 import ipaddress
-import logging
+import subprocess
 
 
-allowed_sni_port = set([
-    ('mtalk.google.com', 5228),
-    ('proxy-safebrowsing.googleapis.com', 80),
-    ('courier.push.apple.com', 5223),
-    ('imap.gmail.com', 993),
-])
+class RemoteServerScanner:
```

Comment:
This sounds like we are scanning servers (like nmap). 
Perhaps something like `IpPacketMonitor`?

Then you can have a `monitor()` method to return the generator. That can return an `IpPacketInfo` object, rather than a generic "observation".

Reply:
ok